src/api/app/views/webui2/webui/monitor/_events.html.haml

```diff
@@ -1 +1,33 @@
-%h2 Statistical Plots
+%h2.mt-4 Statistical Plots
+- unless available_arch_list.empty?
+  .d-flex.justify-content-center
+    = form_tag(controller: 'main', action: 'index') do
+      .form-row
+        .col-sm-12.col-md-6.py-2
+          %label Architecture:
+          = select_tag(:architecture_display, options_for_select(available_arch_list, default_architecture), class: 'custom-select')
+        .col-sm-12.col-md-6.py-2
+          %label Timeframe:
+          - options = [['1 day', '24'], ['1 hour', 1], ['1 week', '168'], ['1 month', '720'], ['1 year', '8760']]
+          = select_tag(:time_display, options_for_select(options, '24'), class: 'custom-select')
+
+.row#graphs{ data: { arch_to_show: default_architecture, monitor_path: monitor_events_path, time_to_show: '24' } }
+  .col-md-12.col-lg-6.py-4.text-center
+    #building
+    %p.my-2 Workers
+    .d-flex.justify-content-center.flot_legend#legend-building
+    .plotspinner= image_tag('ajax-loader.gif')
+  .col-md-12.col-lg-6.py-4.text-center
+    #events
+    %p.my-2 Repositories to recalculate
+    .d-flex.justify-content-center.flot_legend#legend-events
+    .plotspinner= image_tag('ajax-loader.gif')
```

Comment:
better use bootstrap spinners - ajax-loader.gif is kind of bento only

Reply:
We have an example for a spinner in the patterns library: https://obs-patterns.netlify.com/atoms/icons.html

Reply:
Thanks both, already changed. :eyes: